### DISCUSSION TITLE
import projects with module.yml not at zephyr/module.yml

### DISCUSSION THREAD

**Discussion by xR3b0rn** [zephyrproject-rtos/zephyr#80553] [Q&A]
How do I have to configure the west.yml of a project which imports a Zephyr module with a module.yml which is not at zephyr/module.yml but at e.g. arch/zephyr/module.yml?

**Comment by nordicjm** [comment]
The module is wrong

**Reply by xR3b0rn** [reply]
I introduced zephyr support to a library. However, the maintainer is not willing to put OS specific files in the top level of the corresponding repo. How can I handle situations like this then?

**Reply by nordicjm** [reply]
Not possible, the folder needs to be at the top level, if they don't want to add it to their repo I suggest you fork the project and add the files to your fork
